src/components/Editor/Resources/ResourceList.vue

```diff
@@ -121,6 +158,45 @@ export default {
 				attendee: resource,
 			})
 		},
+		async loadRoomSuggestions() {
+			if (this.resources.length > 0 || !this.hasAdvancedFilters) {
+				this.suggestedRooms = []
+				return
+			}
+
+			try {
+				logger.info('fetching suggestions for ' + this.attendees.length + ' attendees')
+				const query = {
+					capacity: this.attendees.length,
+				}
+				const results = (await advancedPrincipalPropertySearch(query)).map(principal => {
+					return {
+						commonName: principal.displayname,
+						email: principal.email,
+						calendarUserType: principal.calendarUserType,
+						displayName: principal.displayname ?? principal.email,
+						isAvailable: true,
+						roomAddress: principal.roomAddress,
+						uri: principal.email,
+						organizer: this.$store.getters.getCurrentUserPrincipal,
+					}
+				})
+
+				await checkResourceAvailability(
+					results,
+					this.$store.getters.getCurrentUserPrincipalEmail,
+					this.calendarObjectInstance.eventComponent.startDate,
+					this.calendarObjectInstance.eventComponent.startDate,
```

Comment:
```suggestion
					this.calendarObjectInstance.eventComponent.startDate,
					this.calendarObjectInstance.eventComponent.endDate,
```

Or is this on purpose?